<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuleForm</class>
 <widget class="QWidget" name="RuleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="lineSubject">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Дисциплина</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineTeacher">
     <property name="placeholderText">
      <string>Преподаватель</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineGroup">
     <property name="placeholderText">
      <string>Контингент</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonMore">
     <property name="text">
      <string>Пожелания</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="text">
      <string>Убрать</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
